modules/lang-painless/src/test/java/org/elasticsearch/painless/RegexTests.java

```diff
@@ -44,8 +45,17 @@ public void testPatternAfterReturn() {
         assertEquals(false, exec("return 'bar' ==~ /foo/"));
     }
 
-    public void testSlashesEscapePattern() {
-        assertEquals(true, exec("return '//' ==~ /\\/\\//"));
+    public void testBackslashEscapesForwardSlash() {
+        assertEquals(true, exec("'//' ==~ /\\/\\//"));
```

Comment:
This one blows up if you don't reorder the lexer rules after you switch it to non-greedy.